css/properties/box-flex-group.json

```diff
@@ -0,0 +1,67 @@
+{
+  "css": {
+    "properties": {
+      "box-flex-group": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/box-flex-group",
+          "support": {
+            "chrome": {
+              "version_added": true,
+              "prefix": "-webkit-"
```

Comment:
This was removed from all three flavors of Chrome and both of Opera. See https://www.chromestatus.com/features/5393405823680512

Reply:
Thanks, fixed in 2ce531a0d0b9b2ce288f9aedf16d95c9ddceda7d.